addons/mass_mailing/static/src/js/wysiwyg.js

```diff
@@ -78,6 +135,13 @@ const MassMailingWysiwyg = Wysiwyg.extend({
             this.toolbar.$el.find('#create-link').toggleClass('d-none', true);
         }
     },
+    /**
+     * @override
+     */
+    _processAndApplyColor: function () {
+        this.odooEditor.editable.focus();
```

Comment:
It is not clear to me why this is needed either.

Reply:
It was handling a corner case related to picking a color before typing (on mobile).
It's out of the scope of this PR, I've removed it.